consensus-types/interfaces/signed_execution_payload_header.go

```diff
@@ -0,0 +1,24 @@
+package interfaces
+
+import (
+	field_params "github.com/prysmaticlabs/prysm/v5/config/fieldparams"
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+)
+
+type ROSignedExecutionPayloadHeader interface {
+	Header() (ROExecutionPayloadHeaderEPBS, error)
+	Signature() [field_params.BLSSignatureLength]byte
+	IsNil() bool
+}
+
+type ROExecutionPayloadHeaderEPBS interface {
+	ParentBlockHash() [32]byte
+	ParentBlockRoot() [32]byte
+	BlockHash() [32]byte
+	GasLimit() uint64
+	BuilderIndex() primitives.ValidatorIndex
+	Slot() primitives.Slot
+	Value() primitives.Gwei
+	BlobKzgCommitmentsRoot() [32]byte
```

Comment:
in this case is a root instead of a hash, but while I agree that having a named constant is better in general, this 32 is so pervasive in our codebase that if we ever change the root length anyway we'll have to refactor everything.